### PR TITLE
[CDAP-14380] Style changes to navbar for product dropdown

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Fixes navbar styling based on design inputs
- Fixes webpack common build to use proper for split chunks.

JIRA: https://issues.cask.co/browse/CDAP-14380
Build: https://builds.cask.co/browse/CDAP-URUT114

### CODE REVIEW

File: cdap-ui/app/cdap/components/Header/ProductDropdown/index.js [hunk 1]
```diff
@@ -142,6 +142,11 @@ export default class ProductDropdown extends Component {
           <DropdownToggle caret>
             <div className="secure-mode-icon">
               <IconSVG name="icon-cogs" />
+              <div className="cdap-logo-container">
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Header/ProductDropdown/index.js]: giving this `cdap-logo-container` class seemed odd. Any particular reason?

File: cdap-ui/app/cdap/components/Header/index.js [hunk 2]
```diff
@@ -144,9 +151,16 @@ export default class Header extends Component {
...
+              {
+                /**
+                 * FIXME: Same here as well.
+                 */
+              }
+              <li
+                id="product-dropdown"
+                className={classnames("with-pointer cdap-menu clearfix", {
+                  'admin-view': administrationURL === location.pathname.replace(/\/cdap/, '')
+                })}>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Header/index.js]: nit: `>` on newline

File: cdap-ui/app/cdap/components/Header/ProductDropdown/index.js (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Header/ProductDropdown/index.js]: Hhmm this was reverting old code. Have removed it since its unnecessary.

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-URUT114-3

**Pull request merged**